docs/execution-providers/community-maintained/CANN-ExecutionProvider.md

```diff
@@ -389,4 +389,3 @@ Following ops are supported by the CANN Execution Provider in single-operator In
 Additional operator support and performance tuning will be added soon.
 
 * [Ascend](https://www.hiascend.com/en/)
-* [CANN](https://www.hiascend.com/en/software/cann)
```

Comment:
removing because of broken link?  can update it to https://www.hiascend.com/cann